src/database/seeders/20230310115577-demo-user.js

```diff
@@ -1,8 +1,9 @@
+/* eslint-disable import/no-extraneous-dependencies */
 const { v4: uuidv4 } = require('uuid');
```

Comment:
Can you use es6 syntax for this file as well

Reply:
> Can you use es6 syntax for this file as well

Done!